INSTALL.md

```diff
@@ -49,198 +31,135 @@ To update an existing clone you can use the following commands:
 You can use [Git for Windows](https://git-for-windows.github.io/) to
 clone the repository on Windows.
 
-# Dependencies
+## Backends
+
+LibreSprite can be compiled with two different backends:
+
+1. Allegro backend (Windows, Linux)
```

Comment:
specify that it's outdated

Reply:
What about adding a note that both the Allegro and Skia backends are slated to be replaced soon? I assume we want to use SDL for all platforms for simplicity.

Reply:
Yeah, that's even better